packages/eslint-plugin/src/rules/pkg-renaming.js

```diff
@@ -0,0 +1,36 @@
+let simpleMappings = {
```

Comment:
should we try to fix old `@emotion/css` and removed `emotion-theming` here or would you prefer adding separate rules for those?

Reply:
Yes, definitely, will do that, completely forgot.